spec/lib/expansion_rules_spec.rb

```diff
@@ -81,7 +82,10 @@
     specify { expect(rules.expansion_fields(:placeholder_topical_event)).to eq(default_fields) }
     specify { expect(rules.expansion_fields(:role_appointment)).to eq(role_appointment_fields) }
     specify { expect(rules.expansion_fields(:service_manual_topic)).to eq(service_manual_topic_fields) }
-    specify { expect(rules.expansion_fields(:step_by_step_nav)).to eq(step_by_step_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav, :part_of_step_navs)).to eq(step_by_step_auth_bypass_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav, :related_to_step_navs)).to eq(step_by_step_auth_bypass_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav, :unspecified_link)).to eq(step_by_step_fields) }
```

Comment:
Perhaps swap lines 87 and 88 around, so that all of the step by step link types that need `step_by_step_auth_bypass_fields` are together, and the one without bypass fields is on its own.

I'd even be tempted to explicitly rename the latter to `step_by_step_fields_without_auth_bypass_field` to make it clear what's going on.